nrich-form-configuration-spring-boot-starter/README.md

```diff
@@ -18,62 +21,78 @@ The artifact is published on [Maven Central Repository](https://search.maven.org
 
-The default configuration values in yaml format for easier modification are given bellow:
+The default configuration values are given bellow in a yaml format for easier modification:
 
 ```yaml
-
 nrich.form-configuration:
-  default-converter-enabled: true
-  form-configuration-mapping:
-
+    default-converter-enabled: true
+    form-configuration-mapping:
 ```
 
-The default converter should be enabled unless users want to provide a completely custom implementation. Overriding of specific constraint properties display configuration resolving can be
-done by implementing a `ConstrainedPropertyValidatorConverterService` interface. That interface implementation will be picked up by the module and used before default converter if `supports`
-method returns true.
+The default converter should be enabled unless users want to provide a completely custom implementation. To override the resolving of specific constraint properties display configuration a custom
+implementation of [`ConstrainedPropertyValidatorConverterService`][constrained-property-validator-converter-service-url] interface should be provided. That interface implementation will be picked up
+by the module and used before the default converter if `supports` method returns true.
 
 ### Using the module
 
-Users should specify a form-configuration-mapping property list that will contain mappings between client side form identifier and server side input. An example configuration is given bellow:
+Users should specify a `form-configuration-mapping` property map that will contain mappings between the form identifiers on the client-side and the classes that hold constraints on the server-side.
 
-```yaml
+An example configuration is given bellow:
 
+```yaml
 nrich.form-configuration:
-  default-converter-enabled: true
-  form-configuration-mapping:
-    user.create-form: net.croz.nrich.example.CreateUserRequest
-    user.update-form: net.croz.nrich.example.UpdateUserRequest
-
+    default-converter-enabled: true
+    form-configuration-mapping:
+        user.create-form: net.croz.nrich.example.CreateUserRequest
+        user.update-form: net.croz.nrich.example.UpdateUserRequest
 ```
 
-This module should be used from the client side and for that a REST endpoint is exposed on `nrich/form/configuration/fetch` URL. The endpoint expects a list of formIdList
-(key inside form-configuration-mapping map) for which the `FormConfiguration` list will be returned. `FormConfiguration` contains a list of properties with defined constraints, clients can then
-convert that configuration to specific client configurations and apply them to their forms.
+This module exposes REST endpoint on `nrich/form/configuration/fetch` URL so that the client can retrieve constraint definitions. The endpoint expects a list of form ids
```

Comment:
Maybe add method (POST) here also just for clarity (we missed it before).

Reply:
Done.

Reply:
Don't see this?

Reply:
Sorry, i forgot to push the changes.